### PR TITLE
Fix errors with global config names no more sorrys please

### LINKED ISSUE #9696: The default clear color of viewport desn't rendered.
**Operating system or device - Godot version:**
Godot 3.0 updated to  5a42bbaace7d8

**Issue description:**

The default clear color of viewport desn't rendered.

### PR DESCRIPTION
**Pull request by Geequlim**
Should fix #9696

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Why are there whitespaces around " and / ?

File: platform/iphone/export/export.cpp [hunk 1]
```diff
@@ -81,7 +81,7 @@ void EditorExportPlatformIOS::get_export_options(List<ExportOption> *r_options)
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_package/debug", PROPERTY_HINT_GLOBAL_FILE, "zip"), ""));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_package/release", PROPERTY_HINT_GLOBAL_FILE, "zip"), ""));
 
-	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "application/name"), ""));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "" application / config / name ""), ""));
```
Comment (akien-mga) [inline platform/iphone/export/export.cpp]: That looks bogus. It should be "application/config/name" no? Same for most other changes below.

Comment (Geequlim) [inline platform/iphone/export/export.cpp]: Opoos!
The editor is crasy!

File: platform/iphone/export/export.cpp [hunk 2]
```diff
@@ -81,7 +81,7 @@ void EditorExportPlatformIOS::get_export_options(List<ExportOption> *r_options)
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_package/debug", PROPERTY_HINT_GLOBAL_FILE, "zip"), ""));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_package/release", PROPERTY_HINT_GLOBAL_FILE, "zip"), ""));
 
-	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "application/name"), ""));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "application/config/name"), ""));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "application/info"), "Made with Godot Engine"));
```
Comment (reduz) [inline platform/iphone/export/export.cpp]: This is intended! If you check, it's not a global setting but an export setting.

File: platform/iphone/export/export.cpp [hunk 3]
```diff
@@ -173,10 +173,10 @@ Error EditorExportPlatformIOS::export_project(const Ref<EditorExportPreset> &p_p
 
 	print_line("binary: " + binary_to_use);
 	String pkg_name;
-	if (p_preset->get("application/name") != "")
-		pkg_name = p_preset->get("application/name"); // app_name
-	else if (String(GlobalConfig::get_singleton()->get("application/name")) != "")
-		pkg_name = String(GlobalConfig::get_singleton()->get("application/name"));
+	if (p_preset->get("application/config/name") != "")
```
Comment (reduz) [inline platform/iphone/export/export.cpp]: This is also intended! Check that it's obtained from the preset

File: platform/iphone/export/export.cpp [hunk 4]
```diff
@@ -173,10 +173,10 @@ Error EditorExportPlatformIOS::export_project(const Ref<EditorExportPreset> &p_p
...
+		pkg_name = p_preset->get("application/config/name"); // app_name
+	else if (String(GlobalConfig::get_singleton()->get("application/config/name")) != "")
+		pkg_name = String(GlobalConfig::get_singleton()->get("application/config/name"));
```
Comment (reduz) [inline platform/iphone/export/export.cpp]: These two I totally missed

File: platform/osx/export/export.cpp [hunk 5]
```diff
@@ -89,7 +89,7 @@ void EditorExportPlatformOSX::get_export_options(List<ExportOption> *r_options)
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_package/debug", PROPERTY_HINT_GLOBAL_FILE, "zip"), ""));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_package/release", PROPERTY_HINT_GLOBAL_FILE, "zip"), ""));
 
-	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "application/name"), ""));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "application/config/name"), ""));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "application/info"), "Made with Godot Engine"));
```
Comment (reduz) [inline platform/osx/export/export.cpp]: This is intended! It's not a global setting

File: platform/osx/export/export.cpp [hunk 6]
```diff
@@ -273,10 +273,10 @@ Error EditorExportPlatformOSX::export_project(const Ref<EditorExportPreset> &p_p
 
 	print_line("binary: " + binary_to_use);
 	String pkg_name;
-	if (p_preset->get("application/name") != "")
-		pkg_name = p_preset->get("application/name"); // app_name
-	else if (String(GlobalConfig::get_singleton()->get("application/name")) != "")
-		pkg_name = String(GlobalConfig::get_singleton()->get("application/name"));
+	if (p_preset->get("application/config/name") != "")
+		pkg_name = p_preset->get("application/config/name"); // app_name
```
Comment (reduz) [inline platform/osx/export/export.cpp]: These two are intended, they are a preset

File: platform/osx/export/export.cpp [hunk 7]
```diff
@@ -273,10 +273,10 @@ Error EditorExportPlatformOSX::export_project(const Ref<EditorExportPreset> &p_p
...
+	else if (String(GlobalConfig::get_singleton()->get("application/config/name")) != "")
+		pkg_name = String(GlobalConfig::get_singleton()->get("application/config/name"));
 	else
```
Comment (reduz) [inline platform/osx/export/export.cpp]: Missed those

File: platform/osx/export/export.cpp [hunk 8]
```diff
@@ -482,8 +482,8 @@ Error EditorExportPlatformOSX::export_project(const Ref<EditorExportPreset> &p_p
 
 	print_line("binary: " + binary_to_use);
 	String pkg_name;
-	if (p_preset->get("application/name") != "")
-		pkg_name = p_preset->get("application/name"); // app_name
+	if (p_preset->get("application/config/name") != "")
+		pkg_name = p_preset->get("application/config/name"); // app_name
```
Comment (reduz) [inline platform/osx/export/export.cpp]: Again, these two are intended, it's the preset

File: platform/uwp/export/export.cpp [hunk 9]
```diff
@@ -774,7 +774,7 @@ class EditorExportUWP : public EditorExportPlatform {
 		String architecture = arch == ARM ? "ARM" : arch == X86 ? "x86" : "x64";
 		result = result.replace("$architecture$", architecture);
 
-		result = result.replace("$display_name$", String(p_preset->get("package/display_name")).empty() ? (String)GlobalConfig::get_singleton()->get("application/name") : String(p_preset->get("package/display_name")));
+		result = result.replace("$display_name$", String(p_preset->get("package/display_name")).empty() ? (String)GlobalConfig::get_singleton()->get("application/config/name") : String(p_preset->get("package/display_name")));
 		result = result.replace("$publisher_display_name$", p_preset->get("package/publisher_display_name"));
```
Comment (reduz) [inline platform/uwp/export/export.cpp]: good catch

**Review by reduz — DISMISSED** [review]
some were intended (part of a preset, not a global setting)

**Comment by Geequlim** [comment]
Put some intented properties back.

**Pull request merged**